README.md

```diff
@@ -1 +1,226 @@
+
+	oc create -f - <<API
+	apiVersion: v1
+	kind: PersistentVolume
+	metadata:
+	  name: my_pv
+	spec:
+	  capacity:
+	    storage: 10Gi
+	  accessModes:
+	    - ReadWriteOnce
+	    - ReadWriteMany
+	  persistentVolumeReclaimPolicy: Recycle
+	  nfs:
+	    server: localhost
+	    path: /persistent_storage/pv01
+	API
+
+
+## Deploying
+
+
+### Create a Metrics Project
+
+To prevent unwanted components from accessing the metric's persistent volume claim, it is highly recommended to deploy all the metric components within their own project. Management of the metric components is also easier if they are separated in their own project as well.
+
+For the instructions presented here we are going to do so in a project named `metrics`.
+
+To create a new project called `metrics` you will need to run the following command:
+
+	oc new-project metrics
+
+### Create the Deployer Service Account
+
+A pod is used to setup, confiure and deploy all the various metric components. This deployer pod is run under the `metrics-deployer` service account.
+
+To create the metrics deployer service account, the following command can be run:
+
+	oc create -f - <<API
+	apiVersion: v1
+	kind: ServiceAccount
+	metadata:
+	  name: metrics-deployer
+	secrets:
+	- name: metrics-deployer
+	API
+
+### Service Account Permissions
+
+Note: the following commands assumes you are running in the 'metric' project, if you are running in another project, your service account will need to be updated the service account accordingly. The format follows the following format  `system:serviceaccount:$PROJECT_NAME:$SERVICE_ACCOUNT_NAME`
+
+#### Metrics Deployer Service Account
+
+In order to deploy components within the project, the `metrics-deployer` service account needs to be granted the 'edit' permission.
+
+This can be accomplished by running the following command:
+
+	openshift admin policy add-role-to-user edit \
+          system:serviceaccount:metrics:metrics-deployer
+
+
+#### Heapster Service Account
+
+The heapster component requires accessing the kubernetes node to find all the available nodes as well as accessing the `/stats` endpoint on each of those nodes. This means that the `heapster` service account which requires having the `cluster-reader` permission.
+
+The following command will give the `heapster` service account the required permission:
+
+	openshift admin policy add-cluster-role-to-user cluster-reader \
+          system:serviceaccount:metrics:heapster
+
+
+### Create the Hawkular Deployer Secret
+
+The Hawkular deployer pod can use secrets to be used when configuring and deploying the various components. This is useful for specifying certificates and other configuration options. All of these secrets are optional, if no secret is specified the system will either autogenerate a secret, or use defaults.
```

Comment:
'The Hawkular deployer pod can use secrets when configuring and deploying...'.   
'This is useful....' I don't really understand what is being said with this sentence.